torchx/runner/test/config_test.py

```diff
@@ -105,6 +105,18 @@ def _run_opts(self) -> runopts:
             default=None,
             help="a None list option",
         )
+        opts.add(
+            "d",
+            type_=Dict[str, str],
+            default={"foo": "bar"},
+            help="a dict option",
+        )
+        opts.add(
+            "d_none",
+            type_=Dict[str, str],
+            default=None,
+            help="a None dict option",
```

Comment:
can we add a test that use these opts below to verify parsing is correct?

Reply:
Done